doc/Learn/Markup/Binding101.md

```diff
@@ -147,12 +147,11 @@ public partial class MainPage : Page
 		this.DataContext<MyViewModel>((page, vm) => page
 			.Content(
 				new TextBox()
-					.Assign(out var searchBox)
-					.Text(() => vm.Query)
-					.Name("SearchBox"),
+					.Name(out var searchBox)
+					.Text(() => vm.Query),
 				new TextBox()
 					.Text(x => x
-						.Source("SearchBox")
```

Comment:
I'm not really sure about this change. Maybe this should be `nameof(...)`

Reply:
This code is technically accurate... but we as @Youssef1313 points out not doing `nameof` fundamentally changes what we were demonstrating in the sample code.

Reply:
Thanks guys. So `.Source(nameof(searchBox))` ?